packages/permissions/src/hooks/use-is-authorized/use-is-authorized.spec.tsx

```diff
@@ -18,7 +18,6 @@ type TAllAppliedMenuVisibility = {
   value: boolean;
 };
 type TAllAppliedDataFence = {
-  __typename: 'StoreDataFence';
```

Comment:
Can you explain why this is removed? 🤔

Reply:
because that 'fixes' the Typescript error I told you about in the previous comment, but does not address the original issue which is that only `__typename: 'StoreDataFence'` is not optional in `mc.ts`

Reply:
But what was the original error?

Reply:
there was no original error. when I was adding more test cases I had to add `__typename: 'StoreDataFence'` to all of my applied dataFences which I think is not right.

Reply:
You mean in the tests? I would define it where is needed to be honest, if that's how the types are. The types are generated from the graphql queries -> schema.

Reply:
done that 👍

Reply:
Thanks. Btw, if you want to be less repetitive in tests, you can simply have a `createTestStore` function or something which has some defaults values 😉

Reply:
I already had that 👍 I could add more abstraction but the test cases require me to change the demanded and actual dataFences and permissions.